contracts/tansu/src/contract_tansu.rs

```diff
@@ -9,9 +9,8 @@ impl TansuTrait for Tansu {
     /// * `env` - The environment object
     /// * `admin` - The admin address
     fn __constructor(env: Env, admin: Address) {
```

Comment:
Ok so actually what we need to do here is to initialise as paused by just calling `Self::pause(env, admin, true);` or manually might be just better/simpler
```rust
env.storage()
            .instance()
            .set(&types::DataKey::Paused, &true);
```
Because in the end this is the init and we do not need the event. It's more or less useless.

The require_auth for the admin is sort of useless but does not hurt to have it there I suppose. Because when you deploy that runs atomically and you need to sign. So it's a given that you are authenticated.